docs/user_docs/cli/kbcli_addon.md

```diff
@@ -40,6 +40,7 @@ Addon command.
 * [kbcli addon describe](kbcli_addon_describe.md)	 - Describe an addon specification.
 * [kbcli addon disable](kbcli_addon_disable.md)	 - Disable an addon.
 * [kbcli addon enable](kbcli_addon_enable.md)	 - Enable an addon.
+* [kbcli addon index](kbcli_addon_index.md)	 - Manage custom addon indexes
```

Comment:
Where is the `kbcli_addon_index.md`

Reply:
> Where is the `kbcli_addon_index.md`

Due to the wrong usage, the user documentation was not generated automatically. I have made the adjustments.